report/2.Analysis/Requirements.tex

```diff
@@ -0,0 +1,42 @@
+\section{Requirements}
+In order to be able to conclude upon the success of the project, a fixed set of requirements must be formulated.
+These requirements will serve as a guideline in the conclusion of the report and as markers throughout the project. 
+To formally specify the requirements, the MoSCoW analysis model will be utilized. 
+The MoSCoW model specifies which requirements \textbf{M}ust, \textbf{S}hould, \textbf{C}ould, and \textbf{W}on't be fulfilled to deem the project successful.
+
+Throughout the analysis, the term \textit{target} will refer to respectively a small target, eg{.} a ball with a diameter of approximately 10 centimeters or a large target, eg{.} an inflated balloon.
```

Comment:
I don't think this makes sense here. It feels very disconnected from the rest of the section.